src/app/models/propTypes/mainContent/index.js

```diff
@@ -0,0 +1,10 @@
+import { blocksWithTypes } from '../general';
+import { headlineBlockPropTypes } from '../headline';
+import { textBlockPropTypes } from '../text';
+
+const mainContentPropTypes = blocksWithTypes([
```

Comment:
Haven't moved `data.model` props because [they will be removed](https://github.com/bbc/simorgh/issues/329)

Reply:
This if future optimisation but as you'll be picking up the issue straight after happy with this